src/lib.rs

```diff
@@ -49,6 +49,8 @@ pub mod script;
 mod transaction;
 pub mod slip77;
 
+// re-export bitcoin deps which we re-use
+pub use bitcoin::{bech32, hashes, secp256k1};
```

Comment:
Hmm, this is not super fun. I mean this will show up as `elements::hashes` and `elements::bech32`. Is that intended?

Alternatively, we can do 
```rust
pub mod bitcoin {
    pub use bitcoin::{bech32, hashes, secp256k1};
}
```
Hmm, but that might mess with our internal use of the `bitcoin` crate import

Reply:
I think, just doing `pub extern crate bitcoin` would get us the effect you want.

But my intention had been that we'd have `elements::hashes`, `elements::bech32`, etc. Do you think that's too confusing?

Reply:
I think conceptually, we should be using everything from elements. Only minimal code that deals with PegIn/PegOut should have access to the `bitcoin` crate. 

To this end, I would even propose re-exporting the  `bitcoin::PublicKey` to `elements::PublicKey` and other possible dependencies. Maybe this will end causing more confusion, what do you think?

Reply:
Well, one unfortunate thing is that if we merely re-export, rustc will still treat them as the same type. So if we export `elements::PublicKey` then users will still be able to call functions that expect a `bitcoin::PublicKey`. And actualy in rustc error messages, the path that gets shown is pretty-much random as near as I can tell, which is even more confusing.

Reply:
Yeah that's quite the same as doing `pub extern crate bitcoin;`.

I can ACK this change, though. Seems fine.

Just a gentle reminder of https://github.com/stevenroose/rust-elements (branch `elements2`) is a fork of rust-bitcoin so it actually has all equivalent `elements::Xxx` types that are distinct.

Reply:
If we were to make rust-elements a fork of rust-bitcoin, these crates would be re-exported in the same way, and would still not be distinct from the ones exported from rust-bitcoin.